changes/ocd-4852.sql

```diff
@@ -0,0 +1,138 @@
+INSERT INTO openchpl.attestation_report (report_date, attestation_period_id, developer_count, approved_count, pending_acb_action_count, pending_developer_action_count, no_submission_count, last_modified_sso_user) SELECT '2024-11-15', 7, 434, 433, 0, 0, 1,  '6498c4f8-b0f1-70b5-55de-d84faae73402' WHERE NOT EXISTS (SELECT * FROM openchpl.attestation_report WHERE report_date = '2024-11-15');
+INSERT INTO openchpl.attestation_report (report_date, attestation_period_id, developer_count, approved_count, pending_acb_action_count, pending_developer_action_count, no_submission_count, last_modified_sso_user) SELECT '2024-11-13', 7, 434, 430, 2, 1, 1,  '6498c4f8-b0f1-70b5-55de-d84faae73402' WHERE NOT EXISTS (SELECT * FROM openchpl.attestation_report WHERE report_date = '2024-11-13');
```

Comment:
It looks like we're missing data from 2024-11-14. Was that data not provided? Does the report handle the gap? I'm not sure anything needs to change, but I am curious as to the details

There look to be other missing dates too; same questions about those.

Reply:
Yes - there are gaps in the data that Sharon provided.  The report handles the gaps without issues - there are just no data points for the missing dates and the lines appear flat over those missing dates.